source/github-helpers/index.ts

```diff
@@ -37,6 +37,34 @@ export const getCurrentCommittish = (pathname = location.pathname, title = docum
 	return unslashedCommittish;
 };
 
+export const getCurrentBranchFromFeedLink = (): string | undefined => {
+	// .last needed for #2799
+	const feedLink = select.last('link[type="application/atom+xml"]');
+	if (!feedLink) {
+		return;
+	}
+
+	return new URL(feedLink.href)
+		.pathname
+		.split('/')
+		.slice(4) // Drops the initial /user/repo/route/ part
+		.join('/')
+		.replace(/\.atom$/, '');
+};
+
+export const getCurrentBranchFromFindLink = async (): Promise<string | undefined> => {
+	if (pageDetect.isPRFiles()) {
+		return;
+	}
+
+	const feedLink = select.last('a[data-hotkey="t"]') ?? await elementReady('a[data-hotkey="t"]');
```

Comment:
Why last? Why/**when** would there be 2 links with the same keyboard shortcut pointing to 2 different things?

Reply:
> Also if .last can be removed, then we only need elementReady since it calls `select` internally immediately anyway

Got it.

Reply:
a[data-hotkey="t"] is there first

Reply:
Selector order is irrelevant

Reply:
No I mean that elementReady always returned the hotkey first